engine-tests/src/test/java/org/terasology/world/block/shape/BlockShapeTest.java

```diff
@@ -78,7 +54,7 @@ public void testConvexHull() {
         CollisionShape shape = blockShape.getCollisionShape(Rotation.rotate(Yaw.CLOCKWISE_90));
 
         Assert.assertEquals(shape instanceof ConvexHullShape,true);
-        if(shape instanceof ConvexHullShape){
+        if (shape instanceof ConvexHullShape){
```

Comment:
Very minor but a single space could also have been added at the end so it would go `) {`